bin/edgefirsttst.py

```diff
@@ -0,0 +1,108 @@
+
+import argparse
+import pandas as pd
+from lsst.dax.data_generator import columns
+from lsst.dax.data_generator import DataGenerator
+
+
+def edgeOnlyContainedInComplete(chunk_id, objectCount, visitCount, spec):
+    """ Check if all the edge only rows can be matched with identical rows
+    in the complete table table.
+    """
+
+    success = None
+    dataGen = DataGenerator(spec)
+
+    seed = 1
+    edge_width = 0.017 # degrees
```

Comment:
Again, seed and edge_width to be configurable, via a file?

Reply:
Added to datgen.py configuration files (example_spec.py and fakeGenSpec.py).